webapp/client/src/stories/NoPauschbetragPage.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import NoPauschbetragPage from "../pages/NoPauschbetragPage";
+
+export default {
+  title: "Pages/NoPauschbetrag",
+  component: NoPauschbetragPage,
+};
+
+function Template(args) {
+  return <NoPauschbetragPage {...args} />;
+}
+
+export const Default = Template.bind({});
+Default.args = {
+  stepHeader: {
+    title:
+      "Leider haben Sie keinen Anspruch auf behinderungsbedingte Pauschbeträge.",
```

Comment:
We are still providing the title from the backend for consistency? Or is there another reason?

Reply:
Ther eis a different title whether it's person A (single), Person A (joint taxes) or person B.